manifests/o/orhun/git-cliff/2.4.0/orhun.git-cliff.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: orhun.git-cliff
+PackageVersion: 2.4.0
+InstallerType: zip
+NestedInstallerType: portable
+UpgradeBehavior: install
+ReleaseDate: 2024-06-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8
+- Architecture: x64
+  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8
+- Architecture: arm64
+  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8

```

Comment:
```suggestion
Installers:
- Architecture: x86
   NestedInstallerFiles:
   - RelativeFilePath: git-cliff-2.4.0\git-cliff.exe
  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8
- Architecture: x64
   NestedInstallerFiles:
   - RelativeFilePath: git-cliff-2.4.0\git-cliff.exe
  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8
- Architecture: arm64
   NestedInstallerFiles:
   - RelativeFilePath: git-cliff-2.4.0\git-cliff-mangen.exe
  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.4.0/git-cliff-2.4.0-x86_64-pc-windows-msvc.zip
  InstallerSha256: 787D216A589573917C0279CC53C328D1F64ED88B2E9C4A467647527A207698E8
```

Reply:
@tianyeeT for arm64 you need to declare  git-cliff-2.4.0-aarch64-pc-windows-msvc.zip  instead of x86_64